modules/crm_config/config/install/core.entity_view_display.profile.crm_indiv.default.yml

```diff
@@ -8,20 +8,63 @@ dependencies:
     - field.field.profile.crm_indiv.crm_gender
     - field.field.profile.crm_indiv.crm_name
     - field.field.profile.crm_indiv.crm_photo
+    - image.style.thumbnail
     - profile.type.crm_indiv
   module:
+    - address
+    - datetime
     - image
-_core:
-  default_config_hash: oVteP6aET1VrVUyToPz7ML_xzTatXsKiY95vOsJFRg4
+    - options
 id: profile.crm_indiv.default
 targetEntityType: profile
 bundle: crm_indiv
 mode: default
-content: {  }
-hidden:
-  crm_address: true
-  crm_dob: true
-  crm_email: true
-  crm_gender: true
-  crm_name: true
-  crm_photo: true
+content:
+  crm_address:
+    type: address_default
+    weight: 3
+    region: content
+    label: above
+    settings: {  }
+    third_party_settings: {  }
+  crm_dob:
+    type: datetime_default
+    weight: 5
+    region: content
+    label: inline
+    settings:
+      timezone_override: ''
+      format_type: html_date
```

Comment:
We probably want to move decoupled_auth_crm and decoupled_auth_crm_config to the distribution now.